docs/Protocol/publishBlockStream.md

```diff
@@ -0,0 +1,164 @@
+# Block Node Connect Protocol for `publishBlockStream`
+## Definitions
+
+Block Node
+: A software system intended to store and process a Block Stream.  The API for
+  a Block Node is defined in HIP 1056, among others.
+
+Block Number
+: A monotonically increasing number assigned by consensus to each block produced
+  by the network.
+
+Publisher
+: An entity publishing blocks to a Block Node via the `publishBlockStream` API
+  This is typically a Consensus Node or another Block Node.
+
+Subscriber
+: An entity that subscribes to a verified or unverified Block Stream from a
+  Block Node.
+
+Verified Block
+: A verified block is a block for which a Block Proof is received and for which
+  the TSS signature of the network ledger ID is valid.
+
+## Base Protocol
```

Comment:
The format in which is described the logic bugs me a little, as it's gets a little deep and somewhat not very eye friendly. Although I don't have any better suggestions currently, so we can leave it like this. If we come up with something better in the future, this can be easily changed.

Reply:
I agree, I wasn't able to come up with a better layout after trying a few alternatives.
Short of using a formal language (which would have been much less approachable), nothing seemed to improve the structure in a meaningful way.